app/styles/vars.scss

```
@@ -1,3 +1,10 @@
+// Breakpoints
+
+$breakpoint-xs: 360px;
+$breakpoint-s: 640px;
+$breakpoint-m: 768px;
+$breakpoint-l: 1024px;
```

```suggestion
$breakpoint-xs: 360px;
$breakpoint-s: 640px;
$breakpoint-m: 800px;
$breakpoint-l: 1000px;
```

Thoughts on making these more rounded? the specific values are pretty arbitrary, so personal preference is to make them a little easier to remember

---

I think it's probably that 1024 fits into the "harmonic" scale - for example 64 goes 8 times into 1024.

I would say as well it will make the container width of 1000px a little bit more difficult to fit into a breakpoint of 1000px hahah

But other than, no reason not to! Would suggest we maybe we drop the md breakpoint down to 750?

---

Sweet as let's just go with as is and change later if we need :)